src/open_mower/launch/include/_record.launch

```diff
@@ -22,4 +22,13 @@
     <node if="$(optenv OM_ENABLE_RECORDING_ALL False)" pkg="rosbag" type="record" name="rosbag_record_diag_alle"
           args="record -o /$(env HOME)/all_$(arg prefix) -a" />
 
+    <node name="snapshot" pkg="rosbag_snapshot" type="snapshot" args="" subst_value="true">
+        <rosparam>
+            default_duration_limit: $(optenv OM_SNAPSHOT_DURATION 1200)  # Maximum time difference between newest and oldest message, seconds, overrides -d flag
+            default_memory_limit: $(optenv OM_SNAPSHOT_MEMORY_PER_TOPIC 0.0)    # Maximum memory used by messages in each topic's buffer, MB, override -s flag
+            clear_buffer: false        # No option / true will clear the buffer after writing to bag. False will not clear buffer
+            compression: LZ4           # LZ4, BZ2, uncompressed
+            record_all_topics: true
+        </rosparam>
```

Comment:
I think all the other nodes use one XML tag per parameter. Not saying any way is worse, but I think I'd stick to one style for consistency.

Reply:
Done.